frontend/src/app/containers/App.js

```diff
@@ -100,14 +104,51 @@ class App extends Component {
       }
     });
     this.measurePageview();
+
+    const langs = {};
+
+    function addLang(lang, response) {
+      if (response.ok) {
+        response.text().then(data => {
+          langs[lang] = `${langs[lang] || ''}${data}
+`;
+        });
+      }
+    }
+
+    Promise.all(navigator.languages.map(language =>
+      Promise.all(
+        [
+          fetch(`/static/locales/${language}/app.ftl`).then(response => addLang(language, response)),
+          fetch(`/static/locales/${language}/experiments.ftl`).then(response => addLang(language, response))
+        ]
+      )
+    )).then(() => this.props.setLocalizations(langs));
```

Comment:
I've decided to do this in another PR since the current behavior works and this PR is getting huge. I opened a bug for it: https://github.com/mozilla/testpilot/issues/2609